### ISSUE TITLE
Deletion / Termination of User

### ISSUE THREAD

**Issue by renandres** [dholde/BestFoodApp#36]
User Story: A user can initiate to be removed from the platform in order to terminate all relevant data to him/her.

Acceptance criteria:
->

Data:
-> Consent of deletion

**Comment by jbaumont** [comment]
> Is there a difference between deleting an account and deleting data? i.e someone delete their account and get back to the platform few months later.

**Comment by renandres** [comment]
Yes there is. In case of deleting the data everything is terminated, while if we just delete the account, we might ask the user if we should keep the data, in case he/she wants to re-register on the platform
